apps/alerts/lib/repo.ex

```diff
@@ -38,6 +38,13 @@ defmodule Alerts.Repo do
     |> Store.alerts(now)
   end
 
+  @spec by_stop_id(String.t(), DateTime.t()) :: [Alert.t()]
+  def by_stop_id(stop_id, now \\ DateTime.utc_now()) do
+    stop_id
+    |> Store.alert_ids_for_stop_id()
+    |> Store.alerts(now)
```

Comment:
I was gonna ask if you'd considered inserting the whole alert into the ETS table as opposed to just the  Alert IDs, but I poked around and noticed the other functions in this module follow that pattern as well. I guess having it this way (get IDs, then use them to look up alerts relative to now) makes sure we don't accidentally display alerts prior to now? 🤔 Interesting...

Reply:
Yeah I was following the patterns here mainly.  I do think this could benefit from some sort of in memory db, then we have one alert table to query on.  Instead of generating multiple view types.